tasks/patches/add-sbom-and-push.yaml

```diff
@@ -82,15 +90,33 @@
 
       image_sbom["components"].sort(key=lambda c: get_identifier(c))
 
+      # write the CycloneDX unified SBOM 
       with open("./sbom-cyclonedx.json", "w") as f:
         json.dump(image_sbom, f, indent=4)
 
+      # create and write the PURL unified SBOM
       purls = [{"purl": component["purl"]} for component in image_sbom["components"] if "purl" in component]
       purl_content = {"image_contents": {"dependencies": purls}}
 
       with open("sbom-purl.json", "w") as output_file:
           json.dump(purl_content, output_file, indent=4)
 
+      # create Tekton result containing counting of Java publishers
+      java_publishers = {}
+
+      for component in java_sbom["components"]:
+          publisher = component.get("publisher", None)
+
+          if publisher is None:
+              continue
+          
+          if publisher not in java_publishers.keys():
+              java_publishers[publisher] = 0
+          
+          java_publishers[publisher] += 1
+
+      json.dump(java_publishers, os.getenv("results.SBOM_JAVA_COMPONENTS_COUNT.path"), indent=4)
```

Comment:
this will not work, you need to set environment variable (preferred), or use directly` json.dump(java_publishers, "$(results.SBOM_JAVA_COMPONENTS_COUNT.path"), indent=4)`